### PR TITLE
Add banner to conversation list to filter resolved conversations 

### PR DESCRIPTION
**Pull request by akshatdhabi**
### What do you want to achieve?
- Add a banner on conversations list in chat widget to effectively hide resolved conversations.

image 
image

### CODE REVIEW

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 1]
```diff
@@ -6462,6 +6462,28 @@ body.accesibility :focus {
 	-o-animation: km-slide-in-animation .5s ease .5s 1 forwards;
 	animation: km-slide-in-animation .5s ease .5s 1 forwards;
 }
+.mck-conversation-closed{
```
Comment (devashishmamgain) [inline webplugin/css/app/mck-sidebox-1.0.css]: @akshatdhabi can we change it to 'resolved'

File: webplugin/js/app/kommunicate-ui.js [hunk 2]
```diff
@@ -498,6 +500,40 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
...
+        var showAllBannerHtml = "<div id=\"mck-conversation-filter\"><span id=\"mck-conversation-banner-heading\">".concat(MCK_LABELS['filter.conversation.list'].ACTIVE_CONVERSATIONS, "</span><span id=\"mck-conversation-banner-action\" onclick=\"KommunicateUI.toggleShowClosedConversationsStatus(),KommunicateUI.handleClosedConversationsList()\">").concat(MCK_LABELS['filter.conversation.list'].HIDE_RESOLVED, "</span></div>");
+        var closedConversations = document.getElementsByClassName('mck-conversation-closed').length;
+        var openConversations = document.getElementsByClassName('mck-conversation-open').length;
+        var bannerParent = document.querySelector('.mck-conversation.vis .mck-message-inner');
+        var conversationFilterBanner = document.getElementById('mck-conversation-filter');
+        if (totalConversations !== openConversations && totalConversations !== closedConversations && !conversationFilterBanner && bannerParent) {
+            bannerParent.insertAdjacentHTML('afterbegin', showAllBannerHtml);
+        }else if(totalConversations === closedConversations){
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate-ui.js]: @akshatdhabi code formatting required

File: webplugin/css/app/mck-sidebox-1.0.css (hunk 1 above)
Comment (akshatdhabi) [reply webplugin/css/app/mck-sidebox-1.0.css]: done

File: webplugin/js/app/kommunicate-ui.js (hunk 2 above)
Comment (akshatdhabi) [reply webplugin/js/app/kommunicate-ui.js]: done

File: webplugin/js/app/kommunicate-ui.js [hunk 3]
```diff
@@ -10,9 +10,10 @@ KommunicateUI={
     welcomeMessageEnabled : false,
     leadCollectionEnabledOnWelcomeMessage:false,
     anonymousUser:false,
+    showClosedConversations: false,
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate-ui.js]: @akshatdhabi can we name it Resolved here also and any other new code added

Comment (akshatdhabi) [reply webplugin/js/app/kommunicate-ui.js]: @devashishmamgain  done.

**Pull request merged**